systemd/pyproject.toml

```
@@ -0,0 +1,69 @@
+[project]
+name = "charmlibs-systemd"
+description = "The charmlibs.systemd package."
+readme = "README.md"
+requires-python = ">=3.10"
+authors = [
+    {name="The HPC team at Canonical"},
+]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache Software License",
+    "Intended Audience :: Developers",
+    "Operating System :: POSIX :: Linux",
+    "Development Status :: 5 - Production/Stable",
+]
+dynamic = ["version"]
+dependencies = [
+    # "ops",
+]
+
+[dependency-groups]
+lint = [  # installed for `just lint systemd` (unit, functional, and integration are also installed)
+    # "typing_extensions",
+]
+unit = [  # installed for `just unit systemd`
+    "ops[testing]",
+]
+functional = [  # installed for `just functional systemd`
+]
+integration = []
+
+[project.urls]
```

I guess I need to update the template to add these:
```suggestion
[project.urls]
"Documentation" = "https://documentation.ubuntu.com/charmlibs/reference/charmlibs/systemd/"
```